src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+private String value;
+	
```

Comment:
@potatowagon This looks like a blank line, but it actually contains a single tab character. You can configure eclipse to trim trailing whitespace on save: http://stackoverflow.com/questions/1043433/how-to-auto-remove-trailing-whitespace-in-eclipse